data/data-pipeline/data_pipeline/ipython/aggregate_burden_exploration.ipynb

```diff
@@ -0,0 +1,1432 @@
+       "    <tr>\n",
+       "      <th>Low median household income as a percent of area median income (percentile)</th>\n",
+       "      <td>0.469101</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Percent of households in linguistic isolation (percentile)</th>\n",
+       "      <td>0.814475</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Percent of individuals below 200% Federal Poverty Line, imputed and adjusted (percentile)</th>\n",
+       "      <td>0.718936</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Percent individuals age 25 or over with less than high school degree (percentile)</th>\n",
+       "      <td>0.967060</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Percent of individuals &lt; 100% Federal Poverty Line (percentile)</th>\n",
+       "      <td>0.680461</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                       11618\n",
+       "Percent pre-1960s housing (lead paint indicator...  0.019256\n",
+       "Proximity to Risk Management Plan (RMP) facilit...  0.890407\n",
+       "Proximity to NPL sites (percentile)                 0.816638\n",
+       "Proximity to hazardous waste sites (percentile)     0.492804\n",
+       "Wastewater discharge (percentile)                   0.317463\n",
+       "Diagnosed diabetes among adults aged greater th...  0.411229\n",
+       "Current asthma among adults aged greater than o...  0.838579\n",
+       "Coronary heart disease among adults aged greate...  0.121655\n",
+       "Low life expectancy (percentile)                    0.663824\n",
+       "Expected building loss rate (Natural Hazards Ri...  0.843358\n",
+       "Expected agricultural loss rate (Natural Hazard...       NaN\n",
+       "Expected population loss rate (Natural Hazards ...  0.405054\n",
+       "Energy burden (percentile)                          0.213255\n",
+       "Diesel particulate matter exposure (percentile)     0.739199\n",
+       "Traffic proximity and volume (percentile)           0.278872\n",
+       "PM2.5 in the air (percentile)                       0.604720\n",
+       "Unemployment (percent) (percentile)                 0.407881\n",
+       "Housing burden (percent) (percentile)               0.795927\n",
+       "Low median household income as a percent of are...  0.469101\n",
+       "Percent of households in linguistic isolation (...  0.814475\n",
+       "Percent of individuals below 200% Federal Pover...  0.718936\n",
+       "Percent individuals age 25 or over with less th...  0.967060\n",
+       "Percent of individuals < 100% Federal Poverty L...  0.680461"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "usa[usa[\"GEOID10_TRACT\"] == \"08031008387\"][\n",
```

Comment:
What's the goal of this output? Maybe add a brief comment explaining your thinking?

Reply:
Sure